apluslms_roman/builder.py

```diff
@@ -1,14 +1,20 @@
-from os import environ, getuid, getegid
-from os.path import isdir, join
+import logging
+from os import getuid, getegid
+from os.path import isdir
 
-from apluslms_yamlidator.utils.decorator import cached_property
 from apluslms_yamlidator.utils.collections import OrderedDict
+from apluslms_yamlidator.utils.decorator import cached_property
 
+from apluslms_roman.utils.path_mapping import load_from_env
 from .backends import BACKENDS, BuildTask, BuildStep, Environment
 from .observer import StreamObserver
 from .utils.importing import import_string
 from .utils.translation import _
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
Logging is configured by CLI or GUI, so don't ever touch logging level per file.

Reply:
-v is already taken, maybe `-l` or `-ll`?

Reply:
Circular import will happens if we try to import something form `cil`, so I'm thinking to put logger level to `env`

Reply:
newest changes pused